cvat-canvas/src/typescript/editHandler.ts

```diff
@@ -103,7 +105,7 @@ export class EditHandlerImpl implements EditHandler {
 
         this.editLine = (this.canvas as any).polyline();
         if (this.editData.state.shapeType === 'polyline') {
-            (this.editLine as any).on('drawpoint', (e: CustomEvent): void => {
+            (this.editLine as any).on('drawupdate', (e: CustomEvent): void => {
```

Comment:
Only actual change in the PR. Others are to fix linter issues

Reply:
It is an extremely useful comment.